pgjdbc/src/main/java/org/postgresql/util/StreamWrapper.java

```diff
@@ -117,7 +117,18 @@ protected void finalize() throws IOException {
             // forcibly close it because super.finalize() may keep the FD open, which may prevent
             // file deletion
             close();
-            super.finalize();
+            // javac 13 assumes it can throw Throwable
+            try {
+              super.finalize();
+            } catch (RuntimeException e) {
+              throw e;
+            } catch (Error e) {
+              throw e;
+            } catch (IOException e) {
+              throw e;
+            } catch (Throwable e) {
+              throw new RuntimeException("Unexpected exception from finalize", e);
+            }
```

Comment:
A bit strange solution for this topic, but I saw that it's coming from the upstream. Can we also add https://github.com/pgjdbc/pgjdbc/commit/3ff4197492b4e4e07ba9420380ff4aabb8837131?

Reply:
Thanks for your attention. I've added the `@SuppressWarnings({"deprecation", "removal"})` annotation.